.gitignore

```diff
@@ -4,3 +4,8 @@
 *.ipr
 *.iws
 target/
+jdg-internal-imagestream.json
+decisionserver-internal-imagestream.json
```

Comment:
Can you share this over CE mailing list, so others can try running this as well.